commands/issue.go

```diff
@@ -39,7 +39,15 @@ var (
 `,
 	}
 
+	cmdCreateIssue = &Command{
+		Key:   "create",
+		Run:   createIssue,
+		Usage: "issue create [-m <MESSAGE>|-f <FILE>] [-l <LABELS>]",
```

Comment:
`Usage` should be displayed in cases when you passed an invalid flag, and `Usage` and `Long` are both used for generating documentation for a command.